LLama/Native/SafeLLamaContextHandle.cs

```diff
@@ -341,6 +341,47 @@ static SafeLLamaContextHandle()
         [DllImport(NativeApi.libraryName, CallingConvention = CallingConvention.Cdecl)]
         private static extern int llama_set_adapter_lora(SafeLLamaContextHandle context, IntPtr adapter, float scale);
 
+        /// <summary>
```

Comment:
The llama.h file contains a critical comment which explains some things:

>     // Functions to access the adapter's GGUF metadata scalar values
>     // - The functions return the length of the string on success, or -1 on failure
>     // - The output string is always null-terminated and cleared on failure
>     // - When retrieving a string, an extra byte must be allocated to account for the null terminator
>     // - GGUF array values are not supported by these functions

The part about returns should be put into the `<return>` blocks for all the relevant functions.